src/renderer/components/dock/edit-resource.store.ts

```diff
@@ -111,6 +112,11 @@ export class EditResourceStore extends DockTabStore<EditingResource> {
       dispose();
     });
   }
+
+  setData(tabId: TabId, data: EditingResource) {
+    super.setData(tabId, data);
+    
```

Comment:
redundant new line

Reply:
Hmm, setData should be deleted from there at all